components/elm/src/biogeochem/CNCarbonStateType.F90

```diff
@@ -459,7 +459,7 @@ subroutine InitCold(this, bounds, ratio, c12_carbonstate_vars)
                 if (veg_vp%evergreen(veg_pp%itype(p)) == 1._r8) then
                    this%leafc_patch(p)         = 1._r8 * ratio
                    this%leafc_storage_patch(p) = 0._r8
-                else if (veg_pp%itype(p) >= npcropmin) then ! prognostic crop types
+                else if (iscft(veg_pp%itype(p)) >= 1) then ! prognostic crop types
```

Comment:
the definition of iscft in pftvarcon only covers two types 0 and 1, is this ">=" prepping for a future condition?  If the meaning of iscft expands to be non-binary, we should update the description in pftvarcon when this happens.

Reply:
Nope, this was intended as binary. Now converted to that throughout the code. Thanks for the suggestion.